go.mod

```diff
@@ -1,265 +1,107 @@
 module github.com/kyma-project/compass-manager
 
-go 1.23.1
+go 1.23.4
 
 require (
 	github.com/99designs/gqlgen v0.17.43
 	github.com/google/uuid v1.6.0
 	github.com/kyma-incubator/compass/components/director v0.0.0-20240205145543-05672afc5d6f
-	github.com/kyma-project/lifecycle-manager/api v0.0.0-20240202162014-d6c42e009612
+	github.com/kyma-project/lifecycle-manager/api v0.0.0-20250113095044-41115399d588
 	github.com/matryer/is v1.4.1
-	github.com/onsi/ginkgo/v2 v2.20.2
-	github.com/onsi/gomega v1.34.1
+	github.com/onsi/ginkgo/v2 v2.21.0
+	github.com/onsi/gomega v1.35.1
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.19.1
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
 	github.com/vektah/gqlparser/v2 v2.5.19
 	github.com/vrischmann/envconfig v1.3.0
-	k8s.io/api v0.30.1
-	k8s.io/apimachinery v0.30.1
-	k8s.io/client-go v0.30.1
-	sigs.k8s.io/controller-runtime v0.18.2
+	k8s.io/api v0.31.2
+	k8s.io/apimachinery v0.32.0
```

Comment:
Is it intentional to use different version of libraries from `k8s.io` group?
It seems that we also have a dependabot PR for the `k8s.io` update that is failing https://github.com/kyma-project/compass-manager/pull/240/files

Reply:
Thanks for pointing it out. Seems this was inherited from KLM and requires a manual upgrade to 0.32. Will adjust it.